File: Services/User/src/main/java/org/example/user/model/dtos/Crew.java
package org.example.user.model.dtos;

import lombok.Builder;
import lombok.Data;
import org.example.user.model.entities.Cabinet;
import org.example.user.model.enums.Role;

import java.time.LocalDate;

@Data
@Builder
public class Crew {
    private String nom;
    private String prenom;
    private String email;
    private String pwd;
    private LocalDate datenaiss;
    private Cabinet cabinet;
    private Role role;
}

File: front/src/app/Models/Crew.ts
import { Cabinet } from "./Cabinet";
import { Role } from "./enums";

export class Crew{
    public id!: number;
    public nom!: string;
    public prenom!: string;
    public email!: string;
    public pwd!: string;
    public role!: Role;
    public locked!: boolean;
    public cabinet!: Cabinet;
}